code/game/gamemodes/dynamic/antag_rulesets.dm

```diff
@@ -0,0 +1,171 @@
+#define MINDSHIELDED_JOBS list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Nanotrasen Representative", "Magistrate", "Internal Affairs Agent", "Nanotrasen Navy Officer", "Special Operations Officer", "Trans-Solar Federation General")
```

Comment:
I'd prefer this either be a GLOB or a static list on /datum/ruleset than a macro, and also sorted with one element per line